### ISSUE TITLE
[ICON REQUEST]: DaVinci Resolve

### ISSUE THREAD

**Issue by KayleeDavisGitHub** [devicons/devicon#1445]
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

DaVinci is a competitor to Adobe products and is a technology for editing videos, applying color correction, and otherwise transforming video content. Given the inclusion of Adobe products (Photoshop, Aftereffects), this may fall within scope but may require further discussion.

#### About: 
https://www.blackmagicdesign.com/products/davinciresolve

#### Logo:
https://en.wikipedia.org/wiki/DaVinci_Resolve#/media/File:DaVinci_Resolve_17_logo.svg

**Comment by Snailedlt** [comment]
Thanks for the suggestion :)

Since this is not strictly a development tool, we'll have to (like you said) discuss this further.
ref: https://github.com/devicons/devicon/wiki/What-Icons-Do-We-Accept%3F#special-cases-see-this-discussion-for-more-details

I added the `discussion` and `awaiting triage` labels to the issue for that reason :)

**Comment by Snailedlt** [comment]
I'm not sure exactly what Davinci Resolve does, but it seems like it's mostly editing of videos. I couldn't find much use for it in software development, nor game development, so I'm not sure it is a great fit for devicons.

@BenSouchet I know you've done a fair bit of design and such. Do you think this icon should be added to devicons?

**Comment by KayleeDavisGitHub** [comment]
We're on the same wavelength @Snailedlt, I just saw [Adobe Premiere Pro in the current list](https://github.com/devicons/devicon/tree/master/icons/premierepro) and given that Davinci is a direct competitor to this software, and is the preferred alternative by many to using this software, I thought it fit the scope through association. I did check [the list](https://github.com/devicons/devicon/wiki/What-Icons-Do-We-Accept%3F) and video editors are not specifically defined anywhere - close to a raster graphics editor maybe? 

Definitely worth discussion because if DaVinci is added there are likely 5+ other video editors that I could think of to add, maybe all for good reason, or maybe the best bet is to remove Premiere & clarify the wiki.

**Comment by lunatic-fox** [comment]
Agree with @KayleeDavisGitHub about clarify the wiki.

**Comment by Snailedlt** [comment]
Agreed! But before we can do that, we need to figure out if video editors should be allowed or not. My vote is on disallowing it, since it's not a tool most developers need afaik. Open to discussion though, and ultimately up to the community as a whole to make the decision!

**Comment by lunatic-fox** [comment]
I also vote the same: disallow icons related to video editors.
So I thought in other questions that may appear and some solutions to them.
> *I think this could be applied to image/graphic editors too.*
```mermaid
graph TD;
  q1{{Video editors related icons are disallowed?}}
  y1{{Should we keep the existing related icons?}}
  n1["Create a new category on 'Allowed' in the Wiki."]

  q1 -- Yes --> y1
  q1 -- No --> n1

  y2[Create a list of exceptions in the Wiki.]
  n2["Update 'Not Allowed (not exhaustive)' in the Wiki."]
  n2a[Remove related icons in the next version of Devicon.]
  
  y1 -- Yes --> y2
  y1 -- No --> n2 --> n2a  
```

**Comment by Snailedlt** [comment]
Nice flowchart @lunatic-fox 

I vote for taking the left route... just modify it a bit:
![image](https://user-images.githubusercontent.com/43886029/200500324-6744e10e-9971-4f06-ae28-f3f1fc81062f.png)

So video editors are generally disallowed, we should keep the existing icons for backwards-compatability, and we should update the wiki with a point under Not Allowed, and crucially also make a list of exceptions, or maybe rather make a more clear point under special cases. The point can say something along the lines of: 
> A lot of icons were added before these rules. Even if they DO NOT follow the rules, they should be kept up to date, and should (for backwards-compatability reasons) NOT be removed

**Comment by KayleeDavisGitHub** [comment]
For what it's worth I think this makes sense as well :)

**Comment by NIDNHU** [comment]
can this please be added? i'd like to add this to my readme

**Comment by Snailedlt** [comment]
@NIDNHU sadly it falls out of scope
